Improve xsdir.json creation

jsbull61 opened the pull request:
Modified to print the full path of the xsdir file as well as the number of CPUs used to create xsdir.json.
    
Added comments and defined the variable 'N' in GettingStarted.ipynb.

jsbull61 committed:
Update listing.py

Modified to print the full path of the xsdir file as well as the number of CPUs used to create xsdir.json.

jsbull61 committed:
Update GettingStarted.ipynb

Added comments and defined the variable 'N'

jlconlin commented:
Thanks @jsbull61!

While you were working on this, I did something similar, but forgot to push it up. Take a look at it here: https://github.com/NuclearData/DataListing/tree/xsdir and let me know which one you think is better; perhaps a combination of the two.

jsbull61 committed:
Update ChargedParticle.ipynb

Update ChargedParticle.ipynb to the version in the xsdir branch of the main repository

jsbull61 committed:
Update files to xsdir; catch xsdir not found error

Update files to main repository xsdir branch

Added message if xsdir is not found.

jsbull61 committed:
Update GettingStarted.ipynb

comment out the hard-coded value for 'N'

jsbull61 commented:
I liked your print statement better so I kept that. I also added that statement to ```listing.py``` I also add am error message if the xsdir file wasn't found in both ```listing.py``` and ```GettingStarted.ipynb``` instead of have the code crash.

jlconlin commented on GettingStarted.ipynb:
```diff
@@ -54,12 +62,25 @@
+    "\n",
+    "# The variable N is how many parallel threads are used\n",
+    "# By default, it uses all but one of the available CPUs\n",
+    "    N = max(1, mp.cpu_count()-1)\n",
```
Your indentation here is not consistent; it's hard to tell from this interface. Please fix this and then I think It's ready to go. Although, I am still (informally) testing on my end to make sure everything works properly.

jlconlin replied:
Ah, you're right. Then please also indent the comments.

jsbull61 replied:
Will do.  Do you want the default value of ```xsdirName``` set back to ```xsdir```?

jsbull61 replied:
If ```xsdirName``` is not set here, the cell crashes with a ```name xsdirName is not defined``` error.

jlconlin replied:
For now, just leave it as `xsdir`. I'll make some changes in a future Pull Request.

jsbull61 committed:
Fix comment indentation; set xsdirName to xsdir

jsbull61 committed:
Print the xsdir file in listing.py

Forgot to add the print line for the xsdir file

jsbull61 commented:
This should be ready to merge into the main repository now.
